pyproject.toml

```diff
@@ -9,6 +9,9 @@ include = ["dpypelines/schemas/**/*.json"]
 [tool.poetry.dependencies]
 python = ">=3.9.0,<3.10.0"
 dpytools = { git = "https://github.com/ONSdigital/dp-python-tools.git", branch="develop" }
+behave = "^1.2.6"
```

Comment:
these are dev dependencies, you can move them down to `tool.poetry.group.dev.dependencies` also flask is only inside the docker container so we shouldnt need a package for it.